packages/connect-common/files/coins.json

```diff
@@ -110,7 +110,10 @@
             "bech32_prefix": "tb",
             "blockchain_link": {
                 "type": "blockbook",
-                "url": ["https://tbtc1.trezor.io", "https://tbtc2.trezor.io"]
+                "url": [
```

Comment:
what produced this formatting changes? is it that we changed our linting rules, or is it some personal setup of yours? this is so hard to read :crying_cat_face:

Reply:
It's from `yarn update-coins` script.

Reply:
seems you are right.